app/views/home/postOnlineTestsDashboard.scala.html

```diff
@@ -51,79 +51,141 @@
     }
```

Comment:
What does postOnlineTestsDashboard mean? It is like a dashboard to display the status when all online tests are finished?